### ISSUE TITLE
IframeWrapper in Playground breaks styled-component render

### ISSUE THREAD

**Issue by marcuslindfeldt** [doczjs/docz#984]
## Bug Report

The IframeWrapper in the Playground does not copy over all styled-components generated styles.

When using conditional styles like the following:

```jsx
    <Avatar {...props}>
      {imageSrc ? (
        <Image alt={`${name} avatar`} src={imageSrc}  />
      ) : (
        <Initials>{getInitals(name)}</Initials>
      )}
    </Avatar>
```

the styles for `Image` and `Initials` will not be applied.

The styles are injected to the main html document but not to the iframe.

**Comment by switz** [comment]
Yeah, also seeing this, but only after hot-reloads. Initially styles are there, but on hot reloads only the main `html` doc gets style updates.

**Comment by rakannimer** [comment]
Thanks for reporting ! 

It should be fixed with 2.0.0-rc.8, please feel free to re-open or open a new issue if it's not resolved !
